opencv_apps/CMakeLists.txt

```diff
@@ -70,6 +74,10 @@ catkin_package()
 include_directories( ${catkin_INCLUDE_DIRS} ${OpenCV_INCLUDE_DIRS})
 
 ## Declare a cpp library
+if(${OPENCV_HAVE_OPTFLOW})
```

Comment:
The proper way to check for a variable is ```if(OPENCV_HAVE_OPTFLOW)```. https://cmake.org/cmake/help/v3.0/command/if.html
```If(<variable>)``` is a bit like what is done above with ```OpenCV_VERSION```.